vnext/Scripts/Microsoft.ReactNative.nuspec

```diff
@@ -21,6 +21,8 @@
     <file src="$nugetroot$\x64\Release\Microsoft.ReactNative\Microsoft.ReactNative.xml"   target="lib\uap10.0"/> 
     -->
 
+    <file src="$nugetroot$\Microsoft.ReactNative.Cxx\**" target ="Microsoft.ReactNative.Cxx"/>
```

Comment:
I understand this is just to unblock, but should we put this in a separate nuget package?

reason for asking is:
 - Managed modules will refer to this as well
 - I assume we want to have this automatically added as a source via a .props or .targets file inside the nuget package.

Reply:
Let me know if you are busy and I can do it as well.

---
In reply to: [439121374](https://github.com/microsoft/react-native-windows/pull/5184#discussion_r439121374) [](ancestors = 439121374)

Reply:
I think that absolutely makes sense.  I'll still do this change to unblock.

If you can look at doing the separate nuget that would be great.

Reply:
Sounds good, will do.